### ISSUE TITLE
keyframes 

### ISSUE THREAD

**Issue by Mannaio** [dbtedman/postcss-prefixwrap#17]
This plugin is wrapping also the animation made with keyframes and it is breaking it

Example output CSS

```
@keyframes pulse {
  .my-custom-wrap 0% {
    opacity: 1;
    -webkit-transform: scale(1);
            transform: scale(1); }
  .my-custom-wrap 80% {
    opacity: 0;
    -webkit-transform: scale(2);
            transform: scale(2); }
  .my-custom-wrap 100% {
    opacity: 0;
    -webkit-transform: scale(3);
            transform: scale(3); } }
```

**Comment by dbtedman** [comment]
Hi @Mannaio thanks for reporting this issue. I am going to take a look at the code and see if I can implement a fix.

**Comment by dbtedman** [comment]
Hi @Mannaio this issue has been resolved in [Release 1.2.0](https://github.com/dbtedman/postcss-prefixwrap/releases/tag/1.2.0).
